dumpling/export/sql_type.go

```diff
@@ -64,6 +66,10 @@ func initColTypeRowReceiverMap() {
 	}
 }
 
+func initUsedBinaryFormat(bf BinaryFormat) {
+	usedBinaryFormat = bf
```

Comment:
How about putting this configuration in `*csvOption` like other csv options?
DM will may start several dumpling tasks, I'm afraid there will be a conflict.

Reply:
Good idea, fixed.